akka-http-core/src/main/scala/akka/http/impl/engine/server/HttpServerBluePrint.scala

```diff
@@ -154,10 +155,16 @@ private[http] object HttpServerBluePrint {
 
     val responseRendererFactory = new HttpResponseRendererFactory(serverHeader, responseHeaderSizeHint, log)
 
+    val errorHandler: Throwable ⇒ Unit = {
+      // socket timeouts should not result in errors in the log. See 19058.
+      case timeout: HttpConnectionTimeoutException ⇒ log.debug(s"Closing HttpConnection due to timeout: ${timeout.getMessage}")
+      case t                                       ⇒ log.error(t, "Outgoing response stream error")
```

Comment:
I think this is because failures in the connection flow are at some point swallowed so they do not kill the binding flow (think i have seen such a line)